src/dashboard.js

```diff
@@ -0,0 +1,26 @@
+// 3rd-party imports
```

Comment:
Small thing, Could you make files like this:

```
src/Dashboard/component.js
src/Dashboard/index.js
```

Check the `Root` component for an example! This just makes a tiny `Dashboard` module.